operator/validator/controller.go

```diff
@@ -1181,13 +1187,27 @@ func SetupCommitteeRunners(ctx context.Context, logger *zap.Logger, options vali
 		// Create a committee runner.
 		epoch := options.BeaconNetwork.GetBeaconNetwork().EstimatedEpochAtSlot(slot)
 		valCheck := TempBeaconVoteValueCheckF(options.Signer, slot, options.SSVShare.Share.SharePubKey, epoch) // TODO: (Alan) fix slashing check (committee is not 1 pubkey)
-		crunner := runner.NewCommitteeRunner(options.BeaconNetwork.GetBeaconNetwork(), shares, buildController(spectypes.RoleCommittee, valCheck), options.Beacon, options.Network, options.Signer, options.OperatorSigner, valCheck)
+		crunner := runner.NewCommitteeRunner(
+			networkConfig,
+			options.BeaconNetwork.GetBeaconNetwork(),
+			shares,
+			buildController(spectypes.RoleCommittee, valCheck),
+			options.Beacon,
+			options.Network,
+			options.Signer,
+			options.OperatorSigner,
+			valCheck,
+		)
 		return crunner.(*runner.CommitteeRunner)
 	}
 }
 
 // SetupRunners initializes duty runners for the given validator
-func SetupRunners(ctx context.Context, logger *zap.Logger, options validator.Options) runner.ValidatorDutyRunners {
+func SetupRunners(
+	ctx context.Context,
+	logger *zap.Logger,
+	options validator.Options,
```

Comment:
lets be consistent with `SetupRunners` and `SetupCommitteeRunners` - either pass networkConfig to both functions or in `validator.Options` I guess if `Options` has it we should we it or remove ( if not needed internally)

Reply:
as far as i can see we're not passing networkConfig to runners

but i did remove the redundant validator.Options.Domain field (its already in Options.NetworkConfig.Domain)

Reply:
good catch, thanks